### PR TITLE
Integrate lastest NNS SDK

### PR DESCRIPTION
**Pull request by johnny7861532**
#19 
integrate lastest NNS SDK - getAddress method.

### CODE REVIEW

File: src/helper/nns/nnsService.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+var neon = require('@cityofzion/neon-js')
+var Neon = neon.default
+var wallet = neon.wallet
+let provider = null;
+
+// assign url
+const NNS = (networkId = 'testnet') => {
+  switch (networkId) {
+    case 'mainnet':
+      return '0x0';
```
Comment (PhyrexTsai) [inline src/helper/nns/nnsService.js]: Using ''

File: src/helper/nns/nnsService.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+export const nnsInit = (provider, networkId) => {
+    provider = new Provider(provider);
+    NNS(networkId);
+}
+
+
+export const owner = async (name) => {
+  try {
+    const script = {
+      scriptHash: NNS, // parameter
```
Comment (PhyrexTsai) [inline src/helper/nns/nnsService.js]: scriptHash: NNS(networkId)

File: src/helper/nns/nnsService.js (hunk 1 above)
Comment (johnny7861532) [reply src/helper/nns/nnsService.js]: Already fix at next commit

File: src/helper/nns/nnsService.js (hunk 2 above)
Comment (johnny7861532) [reply src/helper/nns/nnsService.js]: Already fix at next commit

**Pull request merged**